SKILL.tsv

```diff
@@ -5477,12 +5477,12 @@ SKILL_20151022_005476	$Cannon Blast: Enhance
 SKILL_20151022_005477	$Increases the damage dealt on an enemy with [Cannon Blast] by 1% per attribute level.
 SKILL_20151022_005478	$Increases the attack count of [Blessing] by 10 per attribute level.
 SKILL_20151022_005479	$Carve Austras Koks: Maintain
-SKILL_20151022_005480	$Increases the duration instead of range of [Carve Austras Koks] by 10 seconds.
+SKILL_20151022_005480	$Decreases the range of [Carve Austras Koks], but increases its duration by 10 seconds.
 SKILL_20151022_005481	$Carve Goddess Ausrine: Cooldown Decrease
 SKILL_20151022_005482	$Decreases the cooldown of [Carve Goddess Ausrine] by 10 seconds.
 SKILL_20151022_005483	
 SKILL_20151022_005484	
-SKILL_20151022_005485	
+SKILL_20151022_005485	$While [Resist Elements] is in effect, the chance of nullifying fire, ice, lightning, poison, and earth attacks increases by 0.8% per attribute level.
```

Comment:
Increases the chance of nullifying fire, ice, lightning, poison and earth property attacks by 0.08% per attribute level when (or while, I think 'while' is commonly used) [Resist Elementz] is active.